openpype/hosts/maya/plugins/create/create_animation.py

```diff
@@ -6,17 +6,16 @@
     BoolDef,
     TextDef
 )
+from openpype.pipeline.create import HiddenCreator
 
 
-class CreateAnimation(plugin.MayaCreator):
-    """Animation output for character rigs"""
-
-    # We hide the animation creator from the UI since the creation of it
-    # is automated upon loading a rig. There's an inventory action to recreate
-    # it for loaded rigs if by chance someone deleted the animation instance.
-    # Note: This setting is actually applied from project settings
-    enabled = False
+class CreateAnimation(plugin.MayaCreator, HiddenCreator):
```

Comment:
Hierarchy of creators:
```
└ BaseCreator
  ├ Creator
  ├ HiddenCreator
  └ AutoCreator
```
You should choose only one of them as super class. Now you're combining 2 together.

Reply:
There is a quite a bit of code on the `MayaCreator` that we don't want to recreate or keep in sync. Got any suggestion for mixing in the `HiddenCreator` inheritance?

Reply:
Awaiting merge of #5326 5326